### DISCUSSION TITLE
集成MC基岩版

### DISCUSSION THREAD

**Discussion by gukanyu** [Meloong-Git/PCL#320] [提问] [answered]
目前几乎没有启动器同时支持MC Java版和基岩版，如果PCL2能同时支持，将成为有力的优势

**Comment by Big-Cake-jpg** [comment, marked as answer]
PCL2 不会添加对基岩版的任何支持，请使用官方启动器或微软商店。

请看 #81

**Comment by JerryHan3** [comment]
基岩版MC本身就不需要启动器，官启也只是加了一个快速打开基岩版的一个按钮。
而且基岩版是下载时付费，如果PCL2提供下载方式的话，那就真成盗版了。
再者，基岩版是UWP应用，一般只能由微软商店安装，就跟iOS上的App Store一样。让启动器安装UWP应用很复杂，用户自己也得进行系统设置才行。

**Reply by Super-XiaoNian** [reply]
不过提供安装包倒是可以，但龙猫应该不会干。
